src/agent.c

```diff
@@ -436,6 +437,35 @@ int agent_direct_send(juice_agent_t *agent, const addr_record_t *dst, const char
 	return ret;
 }
 
+int agent_direct_send2(juice_agent_t *agent, const agent_stun_entry_t *entry, const char *data,
```

Comment:
The name `agent_direct_send2` is not very explicit, is there a reason for not modifying `agent_direct_send()` since they have the same signature?

Reply:
I modify the parameters of the function, but I don't want to break other code that relies on the original function

Reply:
Oh, you changed `dst` to `entry`, my bad. You could call it something more explicit then, like `agent_transport_send`.